### DISCUSSION TITLE
Help in setting up Mastodon via podman (user session)

### DISCUSSION THREAD

**Discussion by NaheemSays** [mastodon/mastodon#18876] [Q&A]
I am playing around with trying to install mastodon on a Centos 9 VM I have set up on my PC. I am trying to stick to podman and have used https://musings.danlj.org/2022/06/04/mastodon-on-centos-9-derivatives.html as a template.

As it is a local instance I couldnt copy the bit about https proxy, but I modified that to just be plain http for now.

I have gotten as far as creating an admin account via tootctl, which calls out to podman-compose and then I turned off the creation of accounts.

However when I try to access the website, I get an unable to connect from the firefox web browser. If I try wget 127.0.0.1:3000 from a terminal on the server, I get the following response:

`--2022-07-24 12:06:20--  http://127.0.0.1:3000/
Connecting to 127.0.0.1:3000... connected.
HTTP request sent, awaiting response... 403 Forbidden
2022-07-24 12:06:20 ERROR 403: Forbidden.
`

podman logs mastodon_website_1 gives a lot of output, but the very last line is `[ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: 127.0.0.1`

(Full[er] logs attached)

Any idea on how to figure out what to try next?

[podman_logs.txt](https://github.com/mastodon/mastodon/files/9175662/podman_logs.txt)
?

**Comment by ahoffstadt** [comment]
Have you made any progress on that area? I am also interested in using podman to deploy mastodon.

**Reply by NaheemSays** [reply]
I have not tried it yet, but there was an updated version of the blog I linked to above focussed on podman (but still docker-compose):

https://musings.danlj.org/2022/11/12/mastodon-on-centos-9-derivatives-with-podman.html

Its a good starting point but figuring out why it works with docker-compose and not podman-compose still needs to be done.
